pandas/core/generic.py

```diff
@@ -3331,12 +3331,18 @@ def select(self, crit, axis=0):
 
     def reindex_like(self, other, method=None, copy=True, limit=None,
                      tolerance=None):
-        """Return an object with matching indices to myself.
+        """
+        Return an object with matching indices as other object.
+
+        Conform the object to the same index on all axes. Optional
+        filling logic, placing NA/NaN in locations having no value
+        in the previous index. A new object is produced unless the
+        new index is equivalent to the current one and copy=False.
 
         Parameters
         ----------
         other : Object
-        method : string or None
+        method : {None, 'backfill'/'bfill', 'pad'/'ffill', 'nearest'}
```

Comment:
Can you also add an explanation for the parameters?

Reply:
Sure (copy-pasted the .reindex params)